index.js

```diff
@@ -1,15 +1,19 @@
-const { config, thorify, web3, factoryAddress, exchangeAddress, 
-         Framework, Driver, data } = require( "./utils.js");
+const {
+    factoryAddress,
+    Framework,
+    Driver
+} = require( "./utils.js");
```

Comment:
Does `utils.js` exist anymore? I can't find it checked in

Reply:
This is actually leftover code from Kenneth's original code. I was supposed to get rid of this along with utils.js, but I missed it out index.js. My bad, will remove it.